charts/alfresco-connector-hxi/Chart.yaml

```diff
@@ -13,3 +13,8 @@ dependencies:
     repository: https://alfresco.github.io/alfresco-helm-charts/
     tags:
       - ci
+  - name: elasticsearch
+    repository: https://helm.elastic.co
+    version: 7.17.3
+    tags:
+      - ci
```

Comment:
postgres, not elasticsearch

anyway not mandatory until we are able to make a working helm install on kind

Reply:
🤦‍♂️

Reply:
Fixed